src/features/swap/components/SwapOrder/index.tsx

```diff
@@ -31,10 +32,10 @@ export const SellOrder = ({ order }: { order: SwapOrderType }) => {
     buyToken,
     sellAmount,
     buyAmount,
-    executedSellAmount,
-    executedBuyAmount,
+    class: swapClass,
```

Comment:
the class => orderClass change is live on staging. Maybe we should change this before merging?

Reply:
done